README.md

```diff
@@ -51,45 +51,36 @@ Install it
 
     bundle install
 
-### Rails 4.x.x
+Add to your `app/assets/javascripts/application.js` file
 
-Run the generator
-
-    rails generate autocomplete:install
-
-And include autocomplete-rails.js on your layouts
-
-    javascript_include_tag "autocomplete-rails.js"
```

Comment:
I believe these instructions are remnants from the Rails 3.x.x instructions. For Rails 4 which this gem is designed for, you only need what I've left in.